apps/web/tsconfig.json

```diff
@@ -9,10 +9,7 @@
       "@eventuras/markdowninput": ["../../libs/markdowninput"],
       "@eventuras/scribo": ["../../libs/scribo"],
       "@eventuras/smartform": ["../../libs/smartform"],
-      "@eventuras/sdk": ["../../libs/sdk"],
-      "@eventuras/ui": ["../../libs/ui/src"],
-      "@eventuras/ui/*": ["../../libs/ui/src/*"],
-      "@eventuras/utils": ["../../libs/utils"]
```

Comment:
This change confuses me, if these paths are gone, the compiler does not know where to find them (unless they are in node_modules or linked through it)